modeshape-jcr/src/main/java/org/modeshape/jcr/JcrRepository.java

```diff
@@ -1051,12 +1052,15 @@ protected RunningState( JcrRepository.RunningState other,
                         } else {
                             this.clusteringService = ClusteringService.startStandalone(clusterName, clustering.getConfiguration());        
                         }
+                        JGroupsLockingService jgroupsLockingService = new JGroupsLockingService(this.clusteringService.getChannel(), 
+                                                                                                config.getLockTimeoutMillis());
+                        this.lockingService = clustering.useDbLocking() ? 
+                                              new DbLockingService(jgroupsLockingService, RepositoryCache.INITIALIZATION_LOCK,
+                                                                   this.schematicDb) : jgroupsLockingService;
```

Comment:
Ah, never mind. I now see that the `DbLockingService` uses the `JGroupsLockingService` instance.